Editor/MenuActions/Export/ExportAsset.cs

```diff
@@ -98,11 +99,7 @@ public static string ExportWithFileDialog(IList<ProBuilderMesh> meshes, ExportAs
                         res = ExportMesh(assetPath, pb, options.replaceOriginal, true);
 
                         if (options.replaceOriginal)
-                        {
-                            pb.preserveMeshAssetOnDestroy = true;
-                            Undo.DestroyObjectImmediate(pb);
-                        }
-
+                            StripProBuilderScripts.DestroyProBuilderMeshAndDependencies(pb.gameObject, pb, true);
```

Comment:
@gytis-zasciurinskas All the modifications in this files come from a remark from @karl- .
Using Export>Export as mesh or prefab as causing error while performed on a gameobject with a ProBuilder Shape